packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -47,15 +47,16 @@ export default declare(
   (
     api,
     {
-      jsxPragma = "React",
+      jsxPragma = "React.createElement",
```

Comment:
This change is invisible to users since `(jsxPragmaFrag).split(".")[0]` still returns `React`. It is meant to align with https://github.com/babel/babel/blob/df9ee2c7cda177b0a8810344ad7a41ab456b1b38/packages/babel-plugin-transform-react-jsx/src/transform-classic.js#L6-L9